experimental/framework/data-objects/src/signaler/signaler.ts

```diff
@@ -47,7 +51,7 @@ export interface ISignaler {
 	 * @param signalName - The name of the signal
 	 * @param payload - The data to send with the signal
 	 */
-	submitSignal<T>(signalName: string, payload?: Jsonable<T>);
+	submitSignal<T>(signalName: string, payload?: SignalContentType<T>);
```

Comment:
Update `Readme` for Signaler as well. 
experimental\framework\data-objects\src\signaler\README.md